packages/flame/test/sprite_test.dart

```diff
@@ -41,3 +51,27 @@ class _MyComponent extends PositionComponent {
     sprite.render(canvas, position: size / 2, anchor: Anchor.center);
   }
 }
+
+class _MyRasterComponent extends PositionComponent {
```

Comment:
The test involving this should be in a separate file which is named like the component.